app/views/surveys/index.html.haml

```diff
@@ -5,8 +5,11 @@
     - @surveys.each do |survey|
       = link_to survey_path(survey) do
         .survey
+        -if survey.publish == true || admin_signed_in? 
```

Comment:
The - symbol should have one space separating it from code<br>Line contains trailing whitespace